var/spack/repos/builtin/packages/espresso/package.py

```diff
@@ -85,6 +88,14 @@ class Espresso(Package):
     conflicts('+elpa', when='@:5.4.0')
     conflicts('+hdf5', when='@:5.4.0')
 
+    # To build with QMCPACK's hdf5 interface we version 5.3.0
+    # and hdf5
+    conflicts('+qmcpackconv', when='@5.4:')
```

Comment:
ok, simple to fix. Yes, there did not seem to be a syntax for only use this version.